src/tests/system/tests/test_sss_cache.py

```diff
@@ -43,3 +46,69 @@ def test_sss_cache__cache_expire_message(client: Client):
     for cmd in ("sss_cache -U", "sss_cache -G", "sss_cache -E", "sss_cache --user=nonexisting"):
         res = client.host.ssh.run(cmd)
         assert "No domains configured, fatal error!" not in res.stdout
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.cache
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sss_cache__background_refresh(client: Client, provider: GenericProvider):
+    """
+    :title: Ensuring ldb cache data is refreshed correctly
+    :setup:
+        1. Create provider user
+        2. Create provider group
+        3. Create provider netgroup
+    :steps:
+        1. Set "entry_cache_timeout" to 30 and "refresh_expired_interval" to 60
+        2. Start SSSD
```

Comment:
You mention start in the comments but you restart in the code. Which is the good one?